### ISSUE TITLE
Option to scrap only whitelisted metrics

### ISSUE THREAD

**Issue by mimarpe** [criteo/cassandra_exporter#51]
Hi. Thanks a lot for the project!
While trying to optimize the amount of metrics I store and process in Prometheus, I was wondering, is there a way in cassandra-exporter to scrap a list of metrics or metrics families that we know we want, instead of blacklisting the rest?
The whitelist is much easier to make and maintain in my opinion.

Thanks a lot.

**Comment by erebe** [comment]
Hello,
I should be already possible, in the 'maxScrapFrequencyInSec:
  50:
    - .*
'
instead of '.*' just put the metrics that you want to whitelist. The ones not matching the regex will not be scrapped

**Comment by mimarpe** [comment]
Ok thanks a lot, I'll give a try to that.

**Comment by mimarpe** [comment]
Hi again,
just to give you feedback, your solution works, thanks again.

However, you still need to keep the blacklist in the config (even if only with one metric name) to avoid a NPE and the metrics available. But not a problem for me.

```
Exception in thread "main" java.lang.NullPointerException
        at com.criteo.nosql.cassandra.exporter.JmxScraper.<init>(JmxScraper.java:46)
        at com.criteo.nosql.cassandra.exporter.Main.main(Main.java:27)
```
Regards,
    Miguel

**Comment by erebe** [comment]
Thanks for the feedback, I have fixed it 1148b3b
